lib/active_model_serializers/test/schema.rb

```diff
@@ -1,3 +1,5 @@
+require 'minitest/autorun'
```

Comment:
I guess I should remove this and find another way to make it work? Never seen the ✂️ in a review before? I'll start by cutting this line and not pasting it anywhere else.